src/index.js

```diff
@@ -180,14 +167,18 @@ var execCallback = function (req, res) {
   var vroom = spawn(vroomCommand, reqOptions, {shell: true});
 
   // Handle errors.
-  vroom.on('error', function(err) {
-    console.log(now() + ' - ' + err);
+  vroom.on("error", function(err) {
+    var message = ["Unknown internal error", arg1].join(" ");
```

Comment:
What does `arg1` stands for here? (I did not manage to trigger this error and I don't remember if and how I ever did)